test/sportx.test.ts

```diff
@@ -1,54 +1,59 @@
-import { JsonRpcProvider } from "@ethersproject/providers";
+import { JsonRpcProvider, StaticJsonRpcProvider } from "@ethersproject/providers";
 import { Wallet } from "@ethersproject/wallet";
 import { expect } from "chai";
 import "mocha";
 import { INewOrder, IPendingBetsRequest } from "../src";
 import {
-  DEFAULT_MATIC_RPL_URLS,
+  DEFAULT_RPL_URLS,
   Environments,
   TOKEN_ADDRESSES,
   Tokens,
+  CHAIN_IDS,
+  Networks,
 } from "../src/constants";
 import { ISportX, newSportX } from "../src/sportx";
 import {
   convertFromAPIPercentageOdds,
   convertToAPIPercentageOdds,
   convertToTrueTokenAmount,
 } from "../src/utils/convert";
-import { getSidechainNetwork } from "../src/utils/networks";
+import { getNetwork } from "../src/utils/networks";
 
 // tslint:disable no-string-literal
-
 const TEST_MNEMONIC =
   process.env.TEST_MNEMONIC ||
-  "elegant execute say gain evil afford puppy upon amateur planet lunar pen";
+  // "elegant execute say gain evil afford puppy upon amateur planet lunar pen";
+  "course insect veteran exchange speed goddess drop turn narrow nut enhance agree riot believe release";
```

Comment:
Why is this changed?

Reply:
I thought I would create a new one for SX & have some test SX in it for trying.